frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,76 @@
+import "./EventCard.scss";
+
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+} from "@material-ui/core";
+import React, { BaseSyntheticEvent, ReactElement, useState } from "react";
+
+import { EventParticipation } from "../../../../graphql";
+import useAvatarById from "../../../../hooks/useAvatarById";
+import { useTranslation } from "react-i18next";
+
+export interface EventCardProps {
+  key: string;
+  event: EventParticipation;
+}
+
+export default function EventCard({ event }: EventCardProps): ReactElement {
+  const { t } = useTranslation("common");
+  const avatar = useAvatarById(event.eventId);
+  const [raised, setRaised] = useState<boolean>(false);
+
+  const onMouseOverChange = (_: BaseSyntheticEvent) => {
+    setRaised(!raised);
+  };
+
+  const onButtonClick = () => {
+    // TODO navigate to event details base on event type
+  };
+
+  return (
+    <>
+      <Card
+        raised={raised}
+        className="EventCard"
+        onMouseOver={onMouseOverChange}
+        onMouseOut={onMouseOverChange}
+      >
+        <div className="eventCardContent">
+          <img width="151" src={`data:image/svg+xml;base64,${btoa(avatar)}`} />
```

Comment:
I guess that we use similar `img` tag in several places and doing the same thing (correct me if I'm wrong). Can we just extract this to separate component that will take some id and based on that generate the avatarjbut this time as JSX instead?